rosdep/python.yaml

```diff
@@ -200,6 +200,10 @@ ipython3:
   debian: [ipython3]
   fedora: [ipython3]
   ubuntu: [ipython3]
+jupyter-nbconvert:
+  arch: [jupyter-nbconvert]
+  debian: [jupyter-nbconvert]
+  ubuntu: [jupyter-nbconvert]
```

Comment:
Unfortunately, `jupyter-nbconvert` isn't available on Ubuntu Xenial, which we are still supporting until next year.  We'll have to make a concession to that:

```
ubuntu:
  '*': [jupyter-nbconvert]
  xenial: null
```

Reply:
Ah, thank you for catching it! :)